gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFSInputStream.java

```diff
@@ -138,6 +143,10 @@ public synchronized void seek(long pos) throws IOException {
       channel.position(pos);
     } catch (IllegalArgumentException e) {
       throw new IOException(e);
+    } finally {
+      GhfsTimeStatistic.incrementTimeElapsedMicrosec(
```

Comment:
can this throw exception? If yes, better to handle it since otherwise the original exception will be lost.